.changeset/tiny-vans-watch.md

```diff
@@ -0,0 +1,20 @@
+---
+'@leafygreen-ui/leafygreen-provider': minor
+---
+
+Exposes a `setDarkMode` function form the `useDarkMode` hook.
+Note that the value can still be controlled externally by the prop on LeafyGreenProvider. The value of the last update is used as the context state.
```

Comment:
The last sentence is a little unclear, do you mean that if you use `setDarkMode` then that will be used as the context state?

Reply:
@shaneeza Updated the phrasing. Does it make more sense now?